cornell-dti/carriage-driver
# View switching button

### Summary 






This pull request is the first step towards implementing view switching while in the ride flow

- [x] added schedule button onto ride flow pages that opens greyed out homepage with a car button to return

### Test Plan 

Tested on my device:
1. Start the ride flow
2. Check the schedule from the on the way page 
3. Check the schedule from the pick up page

For the schedule page:
- confirm that rides in progress grid is not shown
- confirm that later rides are greyed out and not tappable
- confirm that if there are no more rides left on this page, the empty ride widget is displayed

### Notes
